swap_meet/item.py

```diff
@@ -1,2 +1,26 @@
+
+
 class Item:
-    pass
+
+    def __init__(self, category='', condition=None):
```

Comment:
Empty string as the default category matches the requirements, but the default condition was specified as 0 for Item and its descendent classes.